app/javascript/packs/application/components/LoginForm.js

```diff
@@ -6,18 +6,15 @@ class LoginForm extends Component {
   static defaultProps = { navigate };
   state = { username: "", password: "" };
 
-  handleSubmit = async event => {
+  handleSubmit = event => {
     event.preventDefault();
-    const result = await this.props.onLogin(
-      this.state.username,
-      this.state.password
-    );
+    this.props.onLogin(this.state.username, this.state.password);
 
-    if (result.ok) {
-      this.props.navigate("/admin");
-    } else {
-      console.error(result.data.error);
-    }
+    // if (result.ok) {
```

Comment:
Are you temporarily commenting this out because you intend to use it in the EXTREMELY NEAR FUTURE?  That's cool.  If not, BE BOLD and get rid of it.  You can always find it in your git history.